src/test/java/seedu/address/logic/commands/EditCommandIntegrationTest.java

```diff
@@ -0,0 +1,325 @@
+ */
+public class EditCommandIntegrationTest {
+
+    private Model model;
+    private Model expectedModel;
+    private ResumeBook resumeBook = new ResumeBookBuilder(TypicalResumeBook.TYPICAL_WITHOUT_GOOGLE).build();
+    private ResumeBook resumeBookCopy = new ResumeBookBuilder(TypicalResumeBook.TYPICAL_WITHOUT_GOOGLE).build();
+
+    /**
+     * A method to set fields in the edit internship descriptor.
+     */
+    public void setEditInternshipDescriptor(EditInternshipDescriptor editInternshipDescriptor,
+                                            Internship sampleEditedInternship) {
+        editInternshipDescriptor.setName(sampleEditedInternship.getName());
+        editInternshipDescriptor.setFrom(sampleEditedInternship.getFrom());
+        editInternshipDescriptor.setTo(sampleEditedInternship.getTo());
+        editInternshipDescriptor.setRole(sampleEditedInternship.getRole());
+        editInternshipDescriptor.setDescription(sampleEditedInternship.getDescription());
+        editInternshipDescriptor.setTags(sampleEditedInternship.getTags());
+    }
+
+    /**
+     * A method to set fields in the edit project descriptor.
+     */
+    public void setEditProjectDescriptor(EditProjectDescriptor editProjectDescriptor, Project sampleEditedProject) {
+        editProjectDescriptor.setName(sampleEditedProject.getName());
+        editProjectDescriptor.setWebsite(sampleEditedProject.getWebsite());
+        editProjectDescriptor.setTime(sampleEditedProject.getTime());
+        editProjectDescriptor.setDescription(sampleEditedProject.getDescription());
+        editProjectDescriptor.setTags(sampleEditedProject.getTags());
+    }
+
+    /**
+     * A method to set fields in the edit skill descriptor.
+     */
+    public void setEditSkillDescriptor(EditSkillDescriptor editSkillDescriptor, Skill sampleEditedSkill) {
+        editSkillDescriptor.setName(sampleEditedSkill.getName());
+        editSkillDescriptor.setLevel(sampleEditedSkill.getLevel());
+        editSkillDescriptor.setTags(sampleEditedSkill.getTags());
+    }
+
+    /**
+     * A method to set fields in the edit note descriptor.
+     */
+    public void setEditNoteDescriptor(EditNoteDescriptor editNoteDescriptor, Note sampleEditedNote) {
+        editNoteDescriptor.setName(sampleEditedNote.getName());
+        editNoteDescriptor.setTime(sampleEditedNote.getTime());
+    }
+
+    /**
+     * A method to set fields in the edit resume descriptor.
+     */
+    public void setEditResumeDescriptor(EditResumeDescriptor editResumeDescriptor, Resume sampleEditedResume) {
+        editResumeDescriptor.setName(sampleEditedResume.getName());
+        editResumeDescriptor.setTags(sampleEditedResume.getTags());
+    }
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(resumeBook, new UserPrefs());
+        expectedModel = new ModelManager(resumeBookCopy, new UserPrefs());
```

Comment:
This line is actually not safe bc resumeBookCopy is being changed.
Before each new ModelManager(new ResumeBookBuilder(TypicalResumeBook.TYPICAL_WITHOUT_GOOGLE).build());